tools/build_variables.py

```diff
@@ -117,7 +117,6 @@
     "torch/csrc/jit/register_special_ops.cpp",
     "torch/csrc/jit/scope.cpp",
     "torch/csrc/jit/script/compiler.cpp",
-    "torch/csrc/api/src/jit.cpp",
```

Comment:
This one disappeared, is it intentional?

Reply:
This was a second problem uncovered by fixing the first; this file was added to both the python extension and libtorch sources, so we got ODR violations.